broadcast/internal/arc/mocks/arc_client_mock_success_nill_query_transaction_response.go

```diff
@@ -0,0 +1,56 @@
+package mocks
+
+import (
+	"context"
+
+	broadcast_api "github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast/broadcast-client-mock/fixtures"
+)
+
+type ArcClientMockNilQueryTransactionResponse struct{}
```

Comment:
the same as above, please make it a bit shorter

Reply:
Changed to shorter name